src/js/App/Sidenav/Navigation/Navigation.scss

```diff
@@ -1,25 +1,26 @@
 .chr-c-app-title {
   color: white;
-  padding: var(--pf-global--spacer--md) var(--pf-global--spacer--md) var(--pf-global--spacer--md) var(--pf-global--spacer--lg);
+  padding: var(--pf-global--spacer--md) var(--pf-global--spacer--sm) var(--pf-global--spacer--md) var(--pf-global--spacer--lg);
   width: 100%;
   border-bottom: var(--pf-global--spacer--xs) solid var(--pf-global--Color--300);
-  text-transform: capitalize;
+  // text-transform: capitalize;
   font-weight: var(--pf-global--FontWeight--semi-bold);
   white-space: normal;
   text-align: left;
-  font-size: var(--pf-global--FontSize--lg);
+  font-size: 17px;
```

Comment:
Is it possible to use some of the PF variables instead?

Reply:
@Hyperkid123 No - this is between PF variable sizes.

Reply:
Do we have to be between the PF sizes?

Reply:
@Hyperkid123 - yes - we can't go bigger and Kendra doesn't want to go smaller (we tested various sizes together)